src/usb/usb_packet.c

```diff
@@ -92,7 +92,7 @@ bool usb_packet_process(const USB_FRAME* frame)
                 ctx, _in_state.data, _in_state.len, _in_state.cmd, _in_state.cid)) {
             // Queue filled and will be sent during usb processing
             _reset_state();
-            return false;
+            return true;
```

Comment:
why this change?

Reply:
This change is to make the return code useful. returning whether "more data is needed" isn't useful anymore. It might've been before we moved the usb communication from interrupts to the main loop.